.github/workflows/ci.yml

```diff
@@ -60,19 +60,18 @@ jobs:
   test:
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest] # TODO add back windows-latest https://github.com/TBD54566975/web5-rs/issues/189
-        rust: [stable, nightly]
+        os: [ubuntu-latest, macos-latest, windows-latest]
     runs-on: ${{ matrix.os }}
     steps:
       - uses: actions/checkout@v4
         with:
           submodules: true
-      - name: Init Hermit
-        uses: cashapp/activate-hermit@v1
-        with:
-          cache: true
+      - name: Install Rust 1.74.0
+        run: rustup toolchain install 1.74.0
+      - name: Set Rust 1.74.0 as default
+        run: rustup default 1.74.0
       - name: Test
-        run: just test
+        run: cargo test --package web5
```

Comment:
y no just no hermit

Reply:
I don't think hermit has windows support, or at least the github action was running into issues on the windows os